Makefile

```diff
@@ -0,0 +1,207 @@
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.30.0
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
```

Comment:
Not sure how Go environments work compared to Python, but is there a recommendation to use a Go "venv", or just use the general user's Go environment?

Reply:
This comes out of the boilerplate code, not sure it is needed by the user.